l3kernel/expl3.dtx

```diff
@@ -144,7 +144,7 @@
 %     \TeX{} programmers to define higher-level commands, and their
 %     idiosyncratic syntax is not at all popular with this community.
 %     Moreover, many of them have names that could be very useful as
-%     document mark-up tags were they not pre-empted as primitives
+%     document mark-up tags were they not pre-emptied as primitives
```

Comment:
No, we mean pre-empted here

Reply:
But the whole phrase is very obscure (whilst being wonderfully erudite, and correct:-)

Reply:
Reverted.